pypeit/core/flexure.py

```diff
@@ -245,13 +246,14 @@ def spec_flex_shift(obj_skyspec, arx_skyspec, mxshft=20):
     msgs.work("Consider taking median first [5 pixel]")
     everyn = obj_skyspec.npix // 20
     bspline_par = dict(everyn=everyn)
-    mask, ct = utils.robust_polyfit(obj_skyspec.wavelength.value, obj_skyspec.flux.value, 3,
-                                    function='bspline', sigma=3., bspline_par=bspline_par)
-    obj_sky_cont = utils.func_val(ct, obj_skyspec.wavelength.value, 'bspline')
+    # TODO - Replace with the other bspline fitter as this no longer works!
```

Comment:
Use pydl.iterfit, which is the function version of the new pydl bspline. This old bspline should be removed everywhere.

Reply:
Fixed by me.